R/render.r

```diff
@@ -354,8 +357,9 @@ pkgdown_footer <- function(data, pkg) {
   list(left = left_final_components, right = right_final_components)
 }
 
-footer_authors <- function(data) {
-  paste0("Developed by ", data$package$authors, ".")
+footer_authors <- function(data, pkg) {
+  text <- pkg$meta$authors$footer$text %||% "Developed by"
+  trimws(paste0(text, " ", data$package$authors, "."))
```

Comment:
Wouldn't it be better to trim `text`? I'm not sure what you're trying to protect against here.

Reply:
yes indeed! I want to protect against two spaces between the text and the authors list.